### PR TITLE
Added GitHub citation file

### PR DESCRIPTION
**Pull request by lobis**
lobis Ok: 76 [](https://gitlab.cern.ch/rest-for-physics/framework/-/commits/lobis-citation)

Adds citation file according to https://docs.github.com/es/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files to allow for "Cite this repository" menu in front page.

image

Data taken from https://doi.org/10.1016/j.cpc.2021.108281, @rest-for-physics/all feel free to add any additional details.

### CODE REVIEW

**Comment by juanangp** [comment]
I have nothing against this PR, but just to let you know you can also cite via zenodo https://zenodo.org/record/6487675#.YnVXwtrP1PY

**Review by juanangp — APPROVED** [review]

**Comment by juanangp** [comment]
> I have nothing against this PR, but just to let you know you can also cite via zenodo https://zenodo.org/record/6487675#.YnVXwtrP1PY

I see, it is linked to the `cff` file, so we have to update this file when we launch a new release? Or this can be somehow automated?

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  given-names: "Susana"
+  affiliation: "Center for Astroparticles and High Energy Physics (CAPA), University of Zaragoza"
+- family-names: "Dafni"
+  given-names: "Theopisti"
+  affiliation: "Center for Astroparticles and High Energy Physics (CAPA), University of Zaragoza"
+- family-names: "Díez-Ibáñez"
+  given-names: "David"
+  affiliation: "Center for Astroparticles and High Energy Physics (CAPA), University of Zaragoza"
+- family-names: "Galán"
+  given-names: "Javier"
```
Comment (jgalan) [inline CITATION.cff:20]:
```suggestion
  given-names: "Javier"
  email: "javier.galan@unizar.es"
  orcid: "https://orcid.org/0000-0001-5673-4431"
```

**Comment by jgalan** [comment]
> > I have nothing against this PR, but just to let you know you can also cite via zenodo https://zenodo.org/record/6487675#.YnVXwtrP1PY
> 
> I see, it is linked to the `cff` file, so we have to update this file when we launch a new release? Or this can be somehow automated?

In principle, there is a zenodo ID to cite all versions 
```
[10.5281/zenodo.4528985](https://doi.org/10.5281/zenodo.4528985).
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,74 @@
+cff-version: 1.2.0
+title: "REST-for-Physics"
+message: "If you use this software, please cite it as below."
+doi: "10.5281/zenodo.6487675"
```
Comment (jgalan) [inline CITATION.cff:4]:
```suggestion
doi: "10.5281/zenodo.4528985"
```

**Review by jgalan — APPROVED** [review]

**Comment by jgalan** [comment]
Before merging I would allow a couple of days to allow other authors to add their ORCID id.

**Pull request merged**